torch/_inductor/triton_helpers.py

```diff
@@ -117,3 +117,51 @@ def _any_combine(a, b):
 @triton.jit
 def any(a, dim):
     return tl.reduce(a, dim, _any_combine)
+
+
+@triton.jit
+def bucketize_binary_search(
+    values,  # 1D tensor
+    offsets_ptr,
+    indexing_dtype,
+    right,  # bool: if true, use intervals closed on the right
+    OFFSETS_SIZE: int,
+    BLOCK_SHAPE,
```

Comment:
It's actually a list/tuple. IDK if it's always 1D or if it can be multi-dimensional sometimes, so I was planning to just pass it in directly like this.

Added a comment to clarify